Formula/seqan@3.rb

```diff
@@ -15,15 +15,25 @@ class SeqanAT3 < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "doxygen" => :build
   depends_on "xz" => :build
+  depends_on "gcc"
```

Comment:
```suggestion
  depends_on "gcc@8"
```

Reply:
I think this is not even necessary. It should work by blacklisting the compilers below, but it doesn't do it for gcc-5 on unix. I don't know why. I have to try it on a linux machine.

Reply:
It should but it does not, had to do the same for https://github.com/brewsci/homebrew-bio/pull/1164